File: Be/property-service/src/main/java/com/property/dto/response/RoomResponse.java
package com.property.dto.response;

import java.util.Set;

import lombok.*;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class RoomResponse {
    String name;

    Boolean isAvailable;

    String rentStatus;
    String roomId;
    String landlordId;
    int currentOccupancy;
    String roomTypeId;
    String apartmentId;
}

File: Fe/client/src/dto/response/roomResponse.ts
import { invoiceType } from "@/dto/response/invoiceResponse";

export type roomResponse = {
	name: string;
	tenants: any;
	rentStatus: string;
	isAvailable: boolean;
	roomId: string;
	currentOccupancy: number;
	apartmentId: string;
	roomTypeId: string
	invoiceResponse: invoiceType[]
}